api/lib/domain/events/handle-pix-plus-certifications-scoring.js

```diff
@@ -18,21 +55,16 @@ async function handlePixPlusCertificationsScoring({
 }) {
   checkEventTypes(event, eventTypes);
   const certificationCourseId = event.certificationCourseId;
-  if (featureToggles.isComplementaryCertificationSubscriptionEnabled) {
-    const hasRunPixPlus = await complementaryCertificationCourseRepository.hasComplementaryCertification({
-      certificationCourseId,
-      complementaryCertificationName: PIX_PLUS_DROIT,
-    });
-    if (!hasRunPixPlus) {
-      return;
-    }
-  }
-
   const certificationAssessment = await certificationAssessmentRepository.getByCertificationCourseId({
     certificationCourseId,
   });
   const certifiableBadgeKeys = certificationAssessment.listCertifiableBadgePixPlusKeysTaken();
-  for (const certifiableBadgeKey of certifiableBadgeKeys) {
+  const allowedToBeScoredBadgeKeys = await _excludeNotAllowedToBeScoredBadgeKeys({
```

Comment:
nitpick:
pourquoi ne pas avoir nommé la fonction:
```suggestion
  const allowedToBeScoredBadgeKeys = await _allowedToBeScoredBadgeKeys({
```

Reply:
C'est vrai qu'on pourrait la nommer dans l'autre sens.